site/content/en/guides/consumer/function/export/circleci/_index.md

```diff
@@ -10,16 +10,19 @@ In this tutorial, you will pull an example blueprint that declares Kubernetes re
 
 ## Before you begin
 
-Before diving into the following tutorial, you need to create a public repo on GitHub, e.g. `function-export-example`.
-
-On your local machine, use `kpt pkg get` to fetch source files of this tutorial:
+Before diving into the following tutorial, you need to create a public repo on GitHub, e.g. `function-export-example`, and clone it into your local machine.
 
 ```shell script
-kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/function-export-blueprint function-export-example
+git clone https://github.com/<USER>/<REPO> function-export-example
```

Comment:
Why?

It was better was previously:

mkdir x
cd x
git init
kpt pkg get
git commit/push

Reply:
It's simpler and more intuitive, I think.